File: 파이썬/Level 1/최소 직사각형.py
def solution(sizes):
    mx, my = 0, 0
    for xy in sizes:
        x,y = min(xy), max(xy)
        if mx * my == 0 :
            mx, my = x, y
        else :
            mx, my = max(mx, x), max(my, y)
    return mx * my

File: 자바스크립트/Level 1/최소 직사각형.js
function solution(sizes) {
    let arr = [0,0]
    sizes.forEach(el=>{
        let [b,s] = el.sort((a,b)=>b-a)
        if(b>arr[0]) arr[0] =b
        if(s>arr[1]) arr[1] =s
    })
    return arr[0]*arr[1]
}